install_tabnine.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Assuming we should even provide the installation script in the first place, it definitely shouldn't be in the root directory. At least in my opinion. For now the best place for it would be in the `misc` directory.